run_test.sh

```diff
@@ -154,7 +161,7 @@ echo 'Adding the test module by directory'
 run_with_expected_output dkms add test/dkms_test-1.0 << EOF
 Creating symlink /var/lib/dkms/dkms_test/1.0/source -> /usr/src/dkms_test-1.0
 EOF
-run_with_expected_output dkms_status_grep_dkms_test << EOF
+run_with_expected_output dkms_status_grep_dkms_module 'dkms_test' << EOF
```

Comment:
This doesn't look quite right. In particular:

The `run_with_expected_xxx` expects a dkms action as `$2` while now we pass the test module. Couple of possible ideas come to mind:
 - explicitly pass action at `$1` + ub the function we `shift` and execute the rest
 - open-code the `dkms_status_grep_dkms_test` function, as far as `run_with_expected...` is conserned
 
Mind you, there could be better options so feel free to opt for something else.